app/models/piece.rb

```diff
@@ -81,4 +81,30 @@ def is_obstructed?(x, y)
       return check_diagonal_obstructions(x, y)
     end
   end
+
+  # Remove the piece from the board
+  def remove_piece(piece) 
+    piece.update_attributes(status: :captured)
+  end
+
+  # Move a piece to the designated location
+  def move_to(x, y)
+
+    if game.piece_in_space?(x, y)
```

Comment:
I thought about that and I don't think the `is_obstructed?` method would be useful here. Since we are just implementing the capture logic, there is no need to check if a move is obstructed. That will come later when we check if a move is valid.

Reply:
Looking at Joseph's comment in Trello: 

Note: This method does not check if a move is valid. We will be using the `valid_move?` method to do that.

It looks like that will be implemented later on.

Reply:
I saw that. I think maybe it would be helpful to discuss with Joe at some point when and where `is_obstructed?` will be used, though, since the `valid_move?` method (from my understanding) is only meant to put some restrictions on how a piece *can* move.